### PR TITLE
WhitelabeledServices: Flatten arrays for multicity and return trip

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: src/booking/types/outputs/services/__tests__/WhitelabeledServices.transportation.test.js [hunk 1]
```diff
@@ -76,22 +76,18 @@ Array [
     "date": 1970-01-01T00:00:00.001Z,
     "location": "OSL",
   },
-  Array [
```
Comment (tbergq) [inline src/booking/types/outputs/services/__tests__/WhitelabeledServices.transportation.test.js]: snaphots are not so good when we don't read them 🙈 

It was trying to tell us this was wrong

Comment (mrtnzlml) [reply src/booking/types/outputs/services/__tests__/WhitelabeledServices.transportation.test.js]: No, I usually don't read them. These tests are not good.

Comment (tbergq) [reply src/booking/types/outputs/services/__tests__/WhitelabeledServices.transportation.test.js]: Should I maybe replace the snapshots here, there is not so huge response anyway

Comment (mrtnzlml) [reply src/booking/types/outputs/services/__tests__/WhitelabeledServices.transportation.test.js]: Yes, you could do that if you want... :)

**Pull request merged**